addons/spectator/ui.hpp

```diff
@@ -84,9 +84,9 @@ class GVAR(display) {
 
             fade = 0.8;
             rows = 1;
-            columns = 2;
-            strings[] = {"$STR_A3_Spectator_Entities","$STR_A3_Spectator_Locations"};
-            values[] = {0,1};
+            columns = 1;
+            strings[] = {"$STR_A3_Spectator_Entities"};
+            values[] = {0};
```

Comment:
Should 0 be replaced with LIST_ENTITIES for consistency? (It doesn't matter for the current state, but if you introduce the meta tab might help)